libsupport/src/TextTracer.cpp

```diff
@@ -55,9 +55,13 @@ BuildText(
   static auto begin = katana::Now();
   auto msec_since_begin = katana::UsSince(begin) / 1000;
 
+  if (!host_data.empty()) {
+    KATANA_WARN_ONCE("To disable TRACE output use flag --disable-trace\n");
```

Comment:
Would it make more sense to put this warning closer to the code that defines `--disable-trace`?

Reply:
that makes sense, I'll move it next to where the flag is handled in the enterprise repo